plugin/core/sessions.py

```diff
@@ -1382,6 +1391,9 @@ def compare_by_string(sb: Optional[SessionBufferProtocol]) -> bool:
     def can_handle(self, view: sublime.View, scheme: str, capability: Optional[str], inside_workspace: bool) -> bool:
         if not self.state == ClientStates.READY:
             return False
+        if self._plugin and self._plugin.should_ignore(view):
+            debug(view, "ignored by plugin", self._plugin.__class__.__name__)
+            return False
```

Comment:
I'm not sure if this is the best place here for this check, because the `can_handle` methods sounds like it might be called more often than necessary. Though the only other reference is in `WindowManager.sessions`, but that seems to be an unused method (?)